CargoMonitor/CargoMonitor.cs

```diff
@@ -583,6 +590,134 @@ public void _handleLimpetLaunchedEvent()
             }
         }
 
+        private void handleCargoDepotEvent(CargoDepotEvent @event)
+        {
+            _handleCargoDepotEvent(@event);
+            writeInventory();
+        }
+
+        private void _handleCargoDepotEvent(CargoDepotEvent @event)
+        {
+            Cargo cargo = new Cargo();
+            HaulageAmount haulageAmount = new HaulageAmount();
+            int amountRemaining = @event.totaltodeliver - @event.delivered;
+
+            switch (@event.updatetype)
+            {
+                case "Collect":
+                    {
+                        cargo = GetCargoWithMissionId(@event.missionid ?? 0);
+                        if (cargo != null)
+                        {
+                            // Cargo instantiated by either 'Mission accepted' event or previous 'WingUpdate' update
+                            haulageAmount = cargo.haulageamounts.FirstOrDefault(ha => ha.id == @event.missionid);
+                            haulageAmount.amount = amountRemaining;
+
+                            //Update commodity definition if intantiated by previous 'WingUpdate' update
+                            if (cargo.commodityDef.edname == "Unknown")
```

Comment:
Update this if we elect to use a more descriptive name...

Reply:
I believe it needs to remain generic to maximize present and future utility. IMO, 'Unknown' is sufficiently descriptive.